tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -379,6 +379,7 @@ def test_run_same_prompt_multiple_times(self, setup):
             include_prompt_logits=True,
             max_tokens=self.num_tokens_generate,
         )
+
```

Comment:
useless diff, you might just run `git checkout origin/main tests/deepsparse/transformers/pipelines/test_text_generation.py` to declutter this PR